pkg/store/database.go

```diff
@@ -101,7 +101,7 @@ func (d *databaseStore) Delete(ctx context.Context, databaseID string) error {
 	err = db.Transaction(func(tx *gorm.DB) error {
 
 		l.Debug("deleting database schema")
-		if err := d.deleteDatabase(tx, databaseID); err != nil {
+		if err := d.deleteDatabaseSchema(tx, databaseID); err != nil {
 			l.Error("failed to delete database schema", zap.Error(err))
 			return meta.NewInternalServerError(err)
```

Comment:
Not a big deal, by why are these status and error messages store in a package called `meta`. It's a little confusing IMHO.

Reply:
It's like.. meta stuff for the api, like status code, error codes, that kind of stuff